cmd/sriovdp/sriov-device-plugin.go

```diff
@@ -29,6 +29,7 @@ import (
 	"time"
 
 	"github.com/golang/glog"
+	"github.com/fsnotify/fsnotify"
```

Comment:
I'm wondering that why glide.yaml is not changed although you add fsnotify in vendor directory...

Did you check glide.yaml in your workspace?

Reply:
Yep, I updated the dependency package manually so glide.yaml is not changed, in my env, glide.yaml doesn't work as expected and it might be a problem we shall fix, but need more testing.